test/e2e/main/main_test.go

```diff
@@ -303,6 +287,33 @@ func TestNodeWithEphemeralStorage(t *testing.T) {
 	}
 }
 
+// Test if single node with a storage class
+func TestNodeWithStorageClass(t *testing.T) {
+	t.Parallel()
+	kindDefaultStorageClass := "standard"
+
+	// Create a resource without passing any config
+	spec := tutils.DefaultSpec(testKube)
+	name := strcase.ToKebab(t.Name())
+	spec.Name = name
+	spec.Spec.Service.Container = &ispnv1.InfinispanServiceContainerSpec{StorageClassName: kindDefaultStorageClass}
+
+	// Register it
+	spec.Labels = map[string]string{"test-name": t.Name()}
+	testKube.CreateInfinispan(spec, tutils.Namespace)
+	defer testKube.CleanNamespaceAndLogOnPanic(tutils.Namespace, spec.Labels)
+	testKube.WaitForInfinispanPods(1, tutils.SinglePodTimeout, spec.Name, tutils.Namespace)
+	testKube.WaitForInfinispanCondition(spec.Name, spec.Namespace, ispnv1.ConditionWellFormed)
+
+	// Ensure a PVCs is bound from the storageClass
+	pvcName := "data-volume-test-node-with-storage-class-0"
+	_, err := testKube.Kubernetes.GetPVC(pvcName, spec.Namespace, context.TODO())
```

Comment:
If you update the `GetPVC` method as suggested, then it's possible to simplify this simply to:

```
testKube.GetPVC("data-volume-test-node-with-storage-class-0", spec.Namespace)
```

All of the subsequent checks can then be removed.

Reply:
added it.